.changeset/lemon-rice-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/celocli': major
```

Comment:
I dont see any Breaking Changes. just additions so this should be a minor not a major 

also contractKit has changest so it will need a changeset too

Reply:
Thanks, updated